pkg/glob/glob.go

```diff
@@ -10,13 +10,18 @@ import (
 
 // TODO: Use native path separators instead of always using /.
 
+type PathInfo struct {
+	Path string
```

Comment:
A reader might question why we can't just use `Info.Name()` here, so document the reason - `Path` is needed to generate paths consistent with the original glob pattern.